cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -51,55 +51,34 @@ def setUp(self):
             display_name='dotted.course.name-2',
         )
 
-    def check_index_and_outline(self, authed_client):
+    def check_courses_on_index(self, authed_client):
         """
-        Test getting the list of courses and then pulling up their outlines
+        Test that the React course listing is present.
         """
         index_url = '/home/'
         index_response = authed_client.get(index_url, {}, HTTP_ACCEPT='text/html')
         parsed_html = lxml.html.fromstring(index_response.content)
-        course_link_eles = parsed_html.find_class('course-link')
-        self.assertGreaterEqual(len(course_link_eles), 2)
-        for link in course_link_eles:
-            self.assertRegexpMatches(
-                link.get("href"),
-                'course/{}'.format(settings.COURSE_KEY_PATTERN)
-            )
-            # now test that url
-            outline_response = authed_client.get(link.get("href"), {}, HTTP_ACCEPT='text/html')
-            # ensure it has the expected 2 self referential links
-            outline_parsed = lxml.html.fromstring(outline_response.content)
-            outline_link = outline_parsed.find_class('course-link')[0]
-            self.assertEqual(outline_link.get("href"), link.get("href"))
-            course_menu_link = outline_parsed.find_class('nav-course-courseware-outline')[0]
-            self.assertEqual(course_menu_link.find("a").get("href"), link.get("href"))
+        courses_tab = parsed_html.find_class('react-course-listing')
+        self.assertEqual(len(courses_tab), 1)
 
-    def test_libraries_on_course_index(self):
+    def test_libraries_on_index(self):
         """
-        Test getting the list of libraries from the course listing page
+        Test that the library tab is present.
         """
-        def _assert_library_link_present(response, library):
+        def _assert_library_tab_present(response):
             """
-            Asserts there's a valid library link on libraries tab.
+            Asserts there's a library tab.
             """
             parsed_html = lxml.html.fromstring(response.content)
-            library_link_elements = parsed_html.find_class('library-link')
-            self.assertEqual(len(library_link_elements), 1)
-            link = library_link_elements[0]
-            self.assertEqual(
-                link.get("href"),
-                reverse_library_url('library_handler', library.location.library_key),
-            )
-            # now test that url
-            outline_response = self.client.get(link.get("href"), {}, HTTP_ACCEPT='text/html')
-            self.assertEqual(outline_response.status_code, 200)
+            library_tab = parsed_html.find_class('react-library-listing')
```

Comment:
Tests creating a library and then verifying that it shows up in the library listing on the studio home page: https://github.com/edx/edx-platform/blob/master/common/test/acceptance/tests/studio/test_studio_home.py#L62

Does not actually click the library link, but could add to that test.